lib/board/pyboard.js

```diff
@@ -1,589 +1,707 @@
 'use babel';
 
-import Config from '../config.js'
+/* eslint-disable no-param-reassign */
```

Comment:
Let's not disable rules for the whole file.
Also, there shouldn't be a reason to disable this particular rule anyway.